citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -99,6 +102,20 @@
         }
     });
 });
+
+    $(document).ready(function() {
+    	var queryParams = window.location.search;
+        if(queryParams)  {
+            const params = new URLSearchParams(queryParams);
+            params.delete('page');
+            queryParams = params.toString();
+        }
+        
+        const x = document.getElementsByClassName("bib-entry");
```

Comment:
is there a reason, you don't use jquery here?

Reply:
also, bad variable name, and why do you use `const` here?

Reply:
No particular reason for not using jquery. I'll change it to jQuery.